SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -82,6 +83,28 @@ public async Task<IActionResult> CreateArticleAsync([FromBody] CreateArticleDTO
             return BadRequest(ValidationMessages.UnableToCreateArticle);
         }
         
+        [Authorize(Roles = "Admin")]
+        [HttpPut("EditArticle")]
+        public async Task<IActionResult> EditArticleAsync([FromBody] CreateArticleDTO adminInput)
+        {
+            ValidationResult validationResult = await _articleValidator.ValidateAsync(adminInput);
+            if (!validationResult.IsValid)
+            {
+                var response = _generateModelStateDictionary.modelStateDictionary(validationResult);
+
+                return ValidationProblem(response);
+            }
+
+            bool editedSuccessfully = await _articleService.EditArticleAsync(adminInput);
+
+            if (editedSuccessfully)
```

Comment:
get rid of True-false, use exception instead

Reply:
This is from `main` and not done by my, should I still implement it?